clientUtils/formatValue.ts

```diff
@@ -10,6 +10,13 @@ export interface TickFormattingOptions {
     showPlus?: boolean
 }
 
+// Used outside this module to figure out if the unit will be joined with the number.
+export function isVeryShortUnit(unit: string): boolean {
+    return ["%", "$", "£"].some((collapsedUnit) =>
+        unit.startsWith(collapsedUnit)
```

Comment:
I'm wondering about the use of `startsWith` here - this means that `% of world population` is also considered a very short unit?

Reply:
Yes, because of this:

https://github.com/owid/owid-grapher/blob/9064f533356fe17d8e8191b680c3d4774db70ed5/clientUtils/formatValue.ts#L34

I don't like the way I implemented it, because the short unit logic here and in the `formatValue` function are completely separate. I didn't use `isVeryShortUnit` in `formatValue` because that handles the two currencies and the percent differently.

Reply:
Ah I see what you mean, whether to have no space in the unit is a different logic from whether it's a short unit. Will change that!